### PR TITLE
Updated documentation with spelling and spaces

### PR DESCRIPTION
**Pull request by niki0712**
## Purpose / Description
_Describe the problem or feature and motivation_

## Fixes
Fixes _Link to the issues._

## Approach
_How does this change address the problem?_

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -52,18 +52,9 @@
...
-      </option>
-      <option name="PACKAGES_IMPORT_LAYOUT">
-        <value>
-          <package name="" alias="false" withSubpackages="true" />
-          <package name="java" alias="false" withSubpackages="true" />
-          <package name="javax" alias="false" withSubpackages="true" />
-          <package name="kotlin" alias="false" withSubpackages="true" />
-          <package name="" alias="true" withSubpackages="true" />
+          <package name="java.util" withSubpackages="false" static="false" />
+          <package name="kotlinx.android.synthetic" withSubpackages="true" static="false" />
```
Comment (david-allison-1) [inline .idea/codeStyles/Project.xml:56]: Hi, this is the only change that you've made, was it intentional?

**Comment by Vaibhavi1707** [comment]
Hi, can you complete the pull request template to explain what spellings or spaces are you fixing? This will make it simpler to understand.

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (niki0712) [reply .idea/codeStyles/Project.xml:56]: Didn't get you, sorry!?

Comment (david-allison-1) [reply .idea/codeStyles/Project.xml:56]: You've only made one change in this PR. Did you mean to change this? It doesn't seem to have anything to do with the title

Comment (niki0712) [reply .idea/codeStyles/Project.xml:56]: Yes I meant to change it I'm sorry if I have chose a wrong title.

**Comment by niki0712** [comment]
@david-allison-1 Yes! I meant to change it. I may have chosen a wrong title. Sorry for the inconvenie

**Comment by niki0712** [comment]
@david-allison-1 Inconvenience*

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Could you explain what this PR is meant to do in the template

**Pull request closed without merging**